prediction_market_agent_tooling/markets/polymarket/polymarket.py

```
@@ -432,6 +437,133 @@ def get_position(
             amounts_current=amounts_current,
         )
 
+    @classmethod
+    def get_positions(
+        cls,
+        user_id: str,
+        liquid_only: bool = False,
+        larger_than: OutcomeToken = OutcomeToken(0),
```

Ruff (0.15.7)

[warning] Do not perform function call `OutcomeToken` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)